docs/docs/userguide/PREREQUISITES.md

```diff
@@ -9,8 +9,9 @@ Its advised that the tooling below is installed to your development environment.
 1. [Terraform](#terraform) 
 2. [Helm v3.3 or later](#helm)
 3. [AWS CLI](#aws-cli)
-4. [Kubectl](#kubectl) (optional)
-5. [Kubernetes cluster monitoring tools](#kubernetes-cluster-monitoring-tools) (optional)
+4. [jq ](#jq)
```

Comment:
```suggestion
4. [jq](#jq)
```

Reply:
Seems that we don't need `jq`, see my comment in the script. We should be ok to use just `aws cli`.